File: src/go/httpGet.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
)

func main() {
	response, _ := http.Get("http://langs.bigsbyspot.org/files/webcall.txt")
	responseBytes, _ := ioutil.ReadAll(response.Body)
	response.Body.Close()
	content := string(responseBytes)
	
	fmt.Println(content)
}

File: src/ts/httpGet.ts
import * as http from "http";
import * as process from "process";

http.get("http://langs.bigsbyspot.org/files/webcall.txt", (response) => {
    var content = "";
    response.on("data", (data) => content += data);
    
    response.on("end", () => {
        process.stdout.write(content);
    });
});